File: 8kyu/well-of-ideas-easy-version/well-of-ideas-easy-version.ts
export function well(x: string[]): string {
  const ideas = x.reduce((acc, curr) => {
    if (curr === "good") acc += 1;

    return acc;
  }, 0);

  if (ideas > 2) return "I smell a series!";
  if (ideas >= 1 && ideas <= 2) return "Publish!";
  return "Fail!";
}

console.log(well(["good", "bad", "bad", "bad", "bad"]));

File: 8kyu/well-of-ideas-easy-version/well-of-ideas-easy-version.go
package main

func Well(x []string) string {
	ideas := 0
	for _, idea := range x {
		if idea == "good" {
			ideas += 1
		}
	}

	if ideas > 2 {
		return "I smell a series!"
	}

	if ideas >= 1 && ideas<=2 {
		return "Publish!"
	}

	return "Fail!"
}